target/linux/ramips/dts/mt7621_dlink_covr-x1860-a1.dts

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "dlink,covr-x1860-a1", "mediatek,mt7621-soc";
+	model = "D-Link COVR-X1860 A1";
+
+	aliases {
+		led-boot = &status_orange;
+		led-failsafe = &status_red;
+		led-running = &status_white;
+		led-upgrade = &status_red;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status_white: power {
```

Comment:
kinda confusing calling it `power` and `status` at the same time.
Perhaps `led-status_white`?

Reply:
indeed it should be `status_white` here to be consistent (although that node name in the dts is technically not used for anything, but it's still confusing).
It's often been a discussion whether to call an LED `power` or `status`, however it should in any case be consistent among LEDs that share the same physical position in the case (i.e. multi-colored)

Reply:
So technically you could just do `led-1`, `led-2`, etc.
Maybe also reorder LEDs by GPIO?

Reply:
How are dependencies to another MR handled usaullay? Should we wait until the other MR is done to apply the changes?